apple/repositories.bzl

```diff
@@ -142,9 +142,9 @@ def apple_rules_dependencies(ignore_version_differences = False):
         http_archive,
         name = "xctestrunner",
         urls = [
-            "https://github.com/bazel-ios/xctestrunner/archive/dfba2695f1ba5930290dd48280d80c06dcdd4ccb.tar.gz",
+            "https://github.com/bazel-ios/xctestrunner/archive/49a4e73a3c7a025a7517155d5ddecb08a552bee1.tar.gz",
```

Comment:
@dostrander I think you'll need to change this to the bumped sha:
https://github.com/bazel-ios/xctestrunner/commit/95dba5afa60647e9d83da8ade9fedace75824002

Reply:
Done!